### ISSUE TITLE
Play button improvements

### ISSUE THREAD

**Issue by hinerm** [imagej/op-finder#1]
- [ ] When trying to run an op like `math.add, it asks for "P2" and then says "P1" (the image) is undefined. So it didn't get preprocessed properly.
- [ ] Should use actual parameter names and not "P1, P2" if possible.. though I assume these are lost  :( still, could do something better (like use the type as the variable name)
- [ ] Should have an `@OUTPUT` for any return values

**Comment by ctrueden** [comment]
> though I assume these are lost :( 

Yep. In Java 8 it is now possible to compile them into the bytecode, so they can be accessed via reflection. We would need to change the `maven-compiler-plugin` configuration in `pom-scijava` to pass the `-parameters` flag, though. This will result in a larger `.class` file footprint, and possibly require more memory in the JVM. Would be worth checking how much more.

See [the relevant documentation](https://docs.oracle.com/javase/tutorial/reflect/member/methodparameterreflection.html) for details.
